evalml/pipelines/components/transformers/samplers/base_sampler.py

```diff
@@ -164,28 +167,16 @@ def __init__(
             parameters=parameters, component_obj=None, random_seed=random_seed
         )
 
-    def fit(self, X, y):
-        """Fits the Oversampler to the data.
-
-        Arguments:
-            X (pd.DataFrame): Training features
-            y (pd.Series): Target features
-
-        Returns:
-            self
-        """
-        super().fit(X, y)
-        self._initialize_oversampler(X, y, self.sampler)
-
-    def _initialize_oversampler(self, X, y, sampler_class):
```

Comment:
Renamed `_initialize_sampler` (also renamed `_initalize_undersampler`) so I could consolidate the `fit` method for both subclasses in the `BaseSampler` class

Reply:
Agreed and done 😁